TUTORIAL.md

```diff
@@ -1416,7 +1416,7 @@ Comments in `.roc` files begin with `##` and will be included in generated docum
 ##     expect x == 2
 ```
 
-Like [Python](https://www.python.org), [Ruby](https://www.ruby-lang.org), [Perl](https://www.perl.org), and [Elixir](https://elixir-lang.org/), Roc also supports inline comments. They begin with `#` instead of `--`, and can also used to add information that won't be included in documentation.
+Like [Python](https://www.python.org), [Ruby](https://www.ruby-lang.org), [Perl](https://www.perl.org), and [Elixir](https://elixir-lang.org/), Roc also supports inline comments. They begin with `#`, and can be used to add information that won't be included in documentation.
```

Comment:
Is this confusing now, with the references to the other languages? Maybe we just remove Python, Ruby, Perl, Elixir altogether? It probably doesn't add much value in this context.

Reply:
Yes, good idea, can you change that?